models/group-model.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 //Database connection
-mongoose.connect('mongodb://localhost/groups', {
+mongoose.connect('mongodb://localhost/message-app', {
```

Comment:
Nice Observation. Please also changed all other root databases to that. So that every one can pull from to  update.